.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
- [ ] in this project, it would be better to set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository and also remember to update your ```linter.yml``` file so that your PR checks will also include ```eslint```. 😎😎
   ## Currently 
   ![image](https://user-images.githubusercontent.com/102757126/216946692-69381b35-5b6c-4da7-8bca-4af7ab595c4b.png)

   _Remember, setting up linters in a project repository is a good practice to ensure the code is written following a consistent style and adheres to best practices. JavaScript and HTML&CSS linters can help identify syntax errors, typo errors, and other issues that might negatively impact the quality and functionality of the code. 👍👍_